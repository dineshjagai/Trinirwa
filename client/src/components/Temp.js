import React from 'react';
import '../App.css';

export default function Temp() {
  return (
    <div className="col-xs-12 col-sm-6 col-md-6 col-lg-6">
      <img src="/api/viewFile/cat1.jpg" alt="" className="img-responsive" />
    </div>

  );
}
