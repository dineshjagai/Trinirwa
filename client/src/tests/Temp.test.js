import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import Temp from '../components/Temp';

test('Test Temp snapshot', () => {
  const component = renderer.create(<Temp />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
