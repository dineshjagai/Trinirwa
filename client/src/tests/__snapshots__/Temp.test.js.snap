// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Temp snapshot 1`] = `
<div
  className="col-xs-12 col-sm-6 col-md-6 col-lg-6"
>
  <img
    alt=""
    className="img-responsive"
    src="/api/viewFile/cat1.jpg"
  />
</div>
`;
